src/main/java/com/gluonhq/substrate/model/Triplet.java

```diff
@@ -28,6 +28,7 @@
 package com.gluonhq.substrate.model;
 
 import com.gluonhq.substrate.Constants;
+import com.gluonhq.substrate.target.*;
```

Comment:
The only issue I see with this is that Triplet.java will depend on all the specific Configurations. Since Triplet.java is used externally, the configurations are exposed externally as well.
Howevere, there is nothing dangerous in exposing this, I believe.

Reply:
Shouldn't we avoid the use of wildcards on imports?

Reply:
@johanvos You're correct, but the public API uses `TargetConfiguration`, so specific ones are effectively hidden. In fact, I think it can be wise to move `TargetConfiguration` out fo that package, then the package does not have to be exported
@jperedadnr this was done by Idea... and I think it is because practically all of the classes from the package are used

Reply:
You can configure Idea to use all the imports?

Reply:
Probably, but it this particular case it was correct :)